docrawl/docrawl_client.py

```diff
@@ -132,9 +132,14 @@ def _execute_function(self, function, function_input=None, timeout=30):
         else:
             docrawl_logger.warning('Browser instance is not active')
 
-    def acquire_browser(self):
+    def acquire_browser(self, driver, in_browser=False, proxy=None):
+        # Need to update browser metadata mainly for proxy
+        self._initialize_browser_metadata(driver=driver, headless=not in_browser, proxy=proxy)
```

Comment:
maybe would make sense to rename to "update_browser_metadata" or "init_or_update_browser_metadata"?